index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link href="style.css" rel="stylesheet">
+</head>
+
+<body>
+  <header>
+    <nav class="navbar">
+      <a class="log">My Logo</a>
+      <nav>
+        <a href="#" class="header-button">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </a>
+      </nav>
+      <div class="nav-links">
+        <ul>
+          <li><a href="#">Home</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <section>
+    <div class="background-img">
+      <div class="row">
+        <div class="main">
+          <h1 class="main-description">
+            <div>I'm Naol</div>
+            <div>Glad to see</div>
+            <div>you</div>
+          </h1>
```

Comment:
Kindly, I suggest you use only text in this section, to make line breaks use '< BR />'